Dockerfile

```diff
@@ -11,23 +24,28 @@ RUN source ~/.cargo/env && \
 
 COPY Cargo.toml .
 COPY Cargo.lock .
-ADD node /creditcoin-node/node
-ADD pallets /creditcoin-node/pallets
-ADD primitives /creditcoin-node/primitives
-ADD runtime /creditcoin-node/runtime
-ADD sha3pow /creditcoin-node/sha3pow
-ADD chainspecs /creditcoin-node/chainspecs
-ADD test /creditcoin-node/test
-RUN apt-get update && apt-get install -y protobuf-compiler
+COPY node /creditcoin-node/node
+COPY pallets /creditcoin-node/pallets
+COPY primitives /creditcoin-node/primitives
+COPY runtime /creditcoin-node/runtime
+COPY sha3pow /creditcoin-node/sha3pow
+COPY chainspecs /creditcoin-node/chainspecs
+COPY test /creditcoin-node/test
 RUN source ~/.cargo/env && cargo build --release
 
-FROM ubuntu:20.04
+FROM ubuntu:22.04
+ENV DEBIAN_FRONTEND=noninteractive
+RUN apt-get update && \
+    apt-get upgrade -y && \
+    apt-get install -y --no-install-recommends ca-certificates && \
+    update-ca-certificates && \
+    rm -rf /var/lib/apt/lists/*
+RUN useradd --home-dir /creditcoin-node --create-home creditcoin
+USER creditcoin
+
 EXPOSE 30333/tcp
 EXPOSE 30333/udp
 EXPOSE 9944 9933 9615
-ENV DEBIAN_FRONTEND=noninteractive
 SHELL ["/bin/bash", "-c"]
-RUN apt-get update && apt-get install -y ca-certificates && update-ca-certificates && rm -rf /var/lib/apt/lists/*
 COPY --from=builder /creditcoin-node/target/release/creditcoin-node /bin/creditcoin-node
-COPY chainspecs .
```

Comment:
So this works fine if you are running on either mainnet or testnet (since the chainspecs for mainnet and testnet are embedded in the creditcoin-node binary for convenience), but this would break our devnet nodes which need `devSpec.json` accessible.

Reply:
Alright. I will restore this commit.